src/components/map-marker/map-marker.tsx

```diff
@@ -68,12 +79,10 @@ export class MapMarker implements GxComponent {
         "GX warning: Can not read 'coords' attribute, default coords set (gx-map-marker)",
         this.element
       );
-
       this.markerInstance = marker([0, 0], {
-        icon: icon({
-          iconAnchor: [12, 41],
-          iconSize: [25, 41],
-          iconUrl: this.iconSrc || this.defaultIcon,
+        icon: divIcon({
+          iconAnchor: [halfIconWidth, this.iconHeight],
+          iconSize: [this.iconWidth, this.iconHeight],
           tooltipAnchor: [0, -28]
```

Comment:
`-28` feels like a magic number. Maybe we should create a constant so we know what it means.
Will it play nicely with icon widths and heights other than `30px`?

Reply:
a great bundle of map-marker issues was fixed in d87d1f4 (```add/MarkerPopUp``` branch)
This issue especially has been fixed in d1b403f